### DISCUSSION TITLE
Example of mixed mesh write using C/C++

### DISCUSSION THREAD

**Discussion by yjhp1016** [CGNS/CGNS#776] [CGNStalk]
Hi, I'm working on writting a mixed mesh  (tet, pyramid, prism and hex) with boundary faces into a CGNS file. I'm using C/C++ I can't find too much examples online, I'm wondering if anyone can help pointing where can I find good example to guide me doing this? Thanks!

**Comment by brtnfld** [comment]
While not in C, the example https://github.com/CGNS/CGNS/tree/develop/src/examples/fortran/Test_mixed_elements provides the basics on how to do this. 

You might also find this code useful:
https://github.com/pvc1989/miniCFD

**Comment by yjhp1016** [comment]
Thanks! It's very helpful! Although the example is in fortran but can
provide a lot of hints about the requirements, layout etc :-)
